mods/kbin-federation-awareness.user.js

```diff
@@ -96,12 +96,12 @@ function kfaGetCss() {
         // Scale 1-10; Default 5 (i.e., 50%); 10 is 50% of 20. 20 * (x * 0.1)
         const defaultScale = 20;
         const setScale = defaultScale * (kfaSettingsScale * 0.1);
-        let fedStyle = ` .comment .data-federated, article .data-federated { display: inline-block; width: ` + setScale + `px; height: ` + setScale + `px; border-radius: 10px; box-shadow: `;
-        let modStyle = ` .comment .data-moderated, article .data-moderated { display: inline-block; width: ` + setScale + `px; height: ` + setScale + `px; border-radius: 10px; box-shadow: `;
-        let homeStyle = ` .comment .data-home, article .data-home { display: inline-block; width: ` + setScale + `px; height: ` + setScale + `px; border-radius: 10px; box-shadow: `;
-        modStyle += `0 0 3px 2px ` + modColor0 + `; background-color: ` + modColor0 + `; margin-right: 4px; }`;
-        fedStyle += `0 0 3px 2px ` + fedColor0 + `; background-color: ` + fedColor0 + `; margin-right: 4px; }`;
-        homeStyle += `0 0 3px 2px ` + homeColor0 + `; background-color: ` + homeColor0 + `; margin-right: 4px; }`;
+        let fedStyle = ` .comment div.data-federated, article .data-federated { display: inline-block; width: ` + setScale + `px; height: ` + setScale + `px; border-radius: 10px; box-shadow: `;
+        let modStyle = ` .comment div.data-moderated, article .data-moderated { display: inline-block; width: ` + setScale + `px; height: ` + setScale + `px; border-radius: 10px; box-shadow: `;
```

Comment:
Would something parameterized like this work? 

```
let dims= "0 0 3px 2px";
let margin = 4
function mod(col){
    let cssBoiler = `{display: inline-block; width: ${setScale}px; height: ${setScale}px; border-radius: 10px; box-shadow: ${dims} ${col} ; background-color: ${col} ; margin-right: ${margin}px; margin-left: ${margin}px; }`;
    let style = `.comment div.data-home, article .data-home ${cssBoiler}`;
    return style
}
const cols = { modColor0: 1, fedColor0: 2, homeColor0: 3 }
let concat
for (const hex in cols){
    concat += mod(`${cols[hex]}`);
}
```

Reply:
Probably. Something to look at in the future.